bin/pre-commit-hook.js

```diff
@@ -123,10 +138,55 @@ function runJSLinter( toLintFiles ) {
 	return lintResult.status;
 }
 
+/**
+ * Runs PHPCS against checked PHP files
```

Comment:
```suggestion
 * Runs PHPCS against checked PHP files. Exits if the check fails.
```
Although I wonder if that's the best behavior, versus having the caller do the exit instead.

Reply:
Yeah, agreed. But since it was like this for quite some time, I would prefer to address it in further PRs